### DISCUSSION TITLE
information about syntax and available functions when using openCV in a python transform

### DISCUSSION THREAD

**Discussion by LaurenzMuessig** [bonsai-rx/bonsai#1537] [General]
Hi all, 
I often use some openCV function(s) within a python transform node - usually to plot something on top of a video frame for visualization. However, I am struggling to find a resource and/or information on which openCV functions I can use and how to call them. Usually I dig through the forum to find someone who has had a similar issue in the past. However, this seems not like the best approach. 

Here is an example:
This is how I import openCV into python:

import clr
clr.AddReference("OpenCV.Net")
from OpenCV.Net import *

within the transform I can use e.g. :
_CV.Threshold(src,dst,thr,max_val,ThresholdTypes.Binary)_

but I can't use:
_CV.CreateImage(size, depth, channels)_

I am not very familliar with either iron python nor openCV so apologies if this is a stupid question. 

Is there anywhere where I can read up on what's allowed and what is not in Bonsai? Thanks a lot for your help.

Cheers

Laurenz

**Comment by bruno-f-cruz** [comment]
This is the documentation website for the OpenCV wrapper used in bonsai: https://horizongir.github.io/opencv.net/

**Reply by LaurenzMuessig** [reply]
Awesome. Exactly what I was looking for. Should've asked about that ages ago. Thanks @bruno-f-cruz . 

Laurenz
